tests/lib/uc20-create-partitions/main.go

```diff
@@ -59,9 +62,12 @@ func (c uc20Constraints) Classic() bool             { return false }
 func (c uc20Constraints) Grade() asserts.ModelGrade { return asserts.ModelSigned }
 
 func main() {
+	if err := logger.SimpleSetup(); err != nil {
```

Comment:
FWIW maybe also add `os.Setenv("SNAPD_DEBUG", "1")` so that we see the logs, unless the test in question is expecting some specific output?

Reply:
well 2 of the task.yaml's have that already defined in the `environment` section so I think it's fine to not include this here IMHO

though I realize now checking there is a third task.yaml which does not have that defined but I think it should